label_config.yml

```diff
@@ -0,0 +1,7 @@
+tripy:
```

Comment:
Why do we have two .yml files?

Reply:
I was experimenting with something. Removed this file now.